app/policies/dashboard_policy.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class DashboardPolicy < ApplicationPolicy
+  def show?
+    collection_creator? || user.depositor_for.any? || user.reviewer_for.any? || user.manages.any?
+  end
+end
```

Comment:
Don't need to account for administrators because that case is already handled by the application policy.

Reply:
Would `.exist?` be faster than `.any?`?

Reply:
@justinlittman `#exists?` and `#any?` both execute the same query, it turns out. I think `user.manages.any?` reads a bit more naturally than `user.manages.exists?`